Comp2023/.pathplanner/settings.json

```diff
@@ -1,6 +1,6 @@
 {
-  "robotWidth": 0.75,
-  "robotLength": 1.0,
+  "robotWidth": 0.8636,
+  "robotLength": 0.86,
```

Comment:
Is there a reason that the width is 0.8636 and the length is 0.86? Our robot is square.

Reply:
I think I meant it to be .8636 when I tried to convert it but on PathPlanner it changed to .86 so I thought it automatically rounded. I can fix that tomorrow.

Reply:
settings.json is actually a text file, so it could just be edited in VS code and saved, too. Don't need to use PathPlanner to change two characters.